diesel/src/macros/mod.rs

```diff
@@ -194,6 +198,24 @@ macro_rules! __diesel_column {
 /// # fn main() {}
 /// ```
 ///
+///
+/// If you want to add documentation for the generated code you could use the
+/// following syntax:
+///
+/// ```
+/// #[macro_use] extern crate diesel;
+///
+/// table! {
+///
+///     /// Table documentation
+///     posts {
+///         id -> Integer => "Id column documentation",
+///         title -> Text => "Title column documentation",
```

Comment:
Can you change this to expect a doc attr as well? I.e.,

```rust
/// Title column documentation
title -> Text,
```

(I'm not sure where exactly attrs are allowed, but here it seems to work: [playground](https://play.rust-lang.org/?gist=a22c8d3556c6e51efe91e88a86004771&version=stable&backtrace=0))